src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -32,11 +32,13 @@
 from azure.mgmt.containerinstance.models import (AzureFileVolume, Container, ContainerGroup, ContainerGroupNetworkProtocol,
                                                  ContainerPort, ImageRegistryCredential, IpAddress, Port, ResourceRequests,
                                                  ResourceRequirements, Volume, VolumeMount, ContainerExecRequestTerminalSize,
-                                                 ContainerGroupIpAddressType, GitRepoVolume, LogAnalytics, ContainerGroupDiagnostics,
-                                                 ContainerGroupNetworkProfile)
+                                                 GitRepoVolume, LogAnalytics, ContainerGroupDiagnostics, ContainerGroupNetworkProfile,
+                                                 ContainerGroupIpAddressType)
+from azure.mgmt.network.models import (Subnet, VirtualNetwork, AddressSpace, Delegation, NetworkProfile,
+                                       ContainerNetworkInterfaceConfiguration, IPConfigurationProfile)
```

Comment:
Network is a multi-API module, so you can't do the imports like this.  In the relevant method, you need to do:

```Python
Subnet, VirtualNetwork = cmd.get_models('Subnet', 'VirtualNetwork')
```

This might be why you are getting the performance issue. Also, you can run the performance check locally with `azdev verify module-load-perf`.